core/Cargo.toml

```diff
@@ -44,7 +44,7 @@ dashmap = { version = "4.0" }
 eyre = "0.6.5"
 futures = { version = "0.3.17", default-features = false, features = ["std", "async-await"] }
 parity-scale-codec = { version = "2", features = ["derive"] }
-rand = "0.7.3"
+rand = "0.8"
```

Comment:
The dependency graph has both of following 2:

* `rand v0.7.3` (a dependency of `ursa v0.3.7`)
* `rand v0.8.4`

Is it okay?

Reply:
shouldn't be an issue, this happens all the time

Reply:
I found no reason to keep rand v0.7.3, but I thought there might be some reason why it has been kept so far. What do you think @eadventurous ?

Reply:
in `futures/Cargo.toml` we already used `rand = 0.8`

Reply:
It's ok to switch it I think